README.md

```diff
@@ -79,6 +79,69 @@ install the protoc plugin for orion
 go get -u github.com/carousell/Orion/protoc-gen-orion
 ```
 
+## protoc-gen-orion
```

Comment:
Considering `protoc-gen-orion` is now an independent tool. Should we move the full explanation to `protoc-gen-orion/README.md`, while only create a link here?